test/framework/integration_framework/iroha_instance.hpp

```diff
@@ -82,16 +192,16 @@ namespace integration_framework {
     }
 
     // config area
-    const std::string block_store_dir_ = "/tmp/block_store";
-    const std::string redis_host_ = std::getenv("IROHA_REDIS_HOST");
-    const size_t redis_port_ = std::stoull(std::getenv("IROHA_REDIS_PORT"));
-    const std::string pg_conn_ = getPostgreCreds();
-    const size_t torii_port_ = 11501;
-    const size_t internal_port_ = 10001;
-    const size_t max_proposal_size_ = 10;
-    const std::chrono::milliseconds proposal_delay_ = 5000ms;
-    const std::chrono::milliseconds vote_delay_ = 5000ms;
-    const std::chrono::milliseconds load_delay_ = 5000ms;
+    std::string block_store_dir_ = "/tmp/block_store";
```

Comment:
I guess we need to move these constants to default parameters of setters.

Reply:
I guess we want get class initialized by default without calling setters.

Reply:
This is another task
https://soramitsu.atlassian.net/browse/IR-810